Installation/cmake/modules/FindSuiteSparse.cmake

```diff
@@ -246,9 +246,9 @@ macro(SuiteSparse_FIND_COMPONENTS )
 		
 		## here we allow to find at least the include OR the lib dir and just warn if one of both missing
 		if(NOT SuiteSparse_${suitesparseCompUC}_INCLUDE_DIR AND NOT SuiteSparse_${suitesparseCompUC}_LIBRARY_RELEASE)
-			set(SuiteSparse_${suitesparseCompUC}_FOUND OFF)
+			set(SuiteSparse_${suitesparseCompUC}_FOUND OFF CACHE INTERNAL "")
 		else()
-			set(SuiteSparse_${suitesparseCompUC}_FOUND ON)
+			set(SuiteSparse_${suitesparseCompUC}_FOUND ON CACHE INTERNAL "")
```

Comment:
It is considered a bad practice to have `<package>_FOUND` set in the cache. Why do you need that?

Reply:
Indeed that is not necessary: the non-cache variable set in `Find<package>.cmake` are accessible from the script calling `find_package(<package>)`.

Reply:
Updated. Thanks!